easybuild/easyconfigs/p/Phantompeakqualtools/Phantompeakqualtools-1.2.2-foss-2021b.eb

```
@@ -0,0 +1,54 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'Phantompeakqualtools'
+version = '1.2.2'
+
+homepage = "https://github.com/kundajelab/phantompeakqualtools"
+
+description = """ It computes informative enrichment and quality measures for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+
+# Download manually from https://github.com/kundajelab/phantompeakqualtools/releases
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b31263b64aefe97bdc4d7fae138f515a7d7a60cd05031d38dd89a10f9ee10cd1']
+
+dependencies = [
+    ('R', '4.2.0'),
+    ('SAMtools', '1.15.1'),
+    ('R-bundle-Bioconductor', '3.15', '-R-4.2.0'),
+]
+
+preinstall_cmd = "rm spp-1.14.tar.gz"
```

Looks like the filename is not entirely correct and should have an underscore instead of hyphen:
```suggestion
preinstall_cmd = "rm spp_1.14.tar.gz"
```